festim/subdomain/surface_subdomain.py

```diff
@@ -22,18 +23,18 @@ def __init__(self, id, x) -> None:
         self.id = id
         self.x = x
 
-    def locate_dof(self, function_space):
+    def locate_dof(self, mesh, fdim):
         """Locates the dof of the surface subdomain within the function space
 
         Args:
-            function_space (dolfinx.fem.FunctionSpace): the function space of
-                the model
+            mesh (dolfinx.mesh.Mesh): the mesh of the simulation
+            fdim (int): the dimension of the model facets
 
         Returns:
             dof (np.array): the first value in the list of dofs of the surface
                 subdomain
         """
-        dofs = fem.locate_dofs_geometrical(
-            function_space, lambda x: np.isclose(x[0], self.x)
+        dofs = dolfinx.mesh.locate_entities_boundary(
+            mesh, fdim, lambda x: np.isclose(x[0], self.x)
```

Comment:
I am pretty sure this now doesn't return dofs but facet indices. Please make the appropriate changes in the namespace